spec/PlatformShFacadeSpec.php

```diff
@@ -81,6 +81,24 @@ function it_finds_the_activity(PlatformClient $client, Project $project, Environ
         $this->waitFor('project_id', 'env', 'sha')->shouldReturn(['the-url']);
     }
 
+    function it_finds_the_activity_for_post_merge_envs(PlatformClient $client, Project $project, Environment $environment, Activity $activity)
+    {
+        $activity->offsetGet('type')->willReturn('environment.push');
+        $activity->offsetExists('parameters')->willReturn(true);
+        $activity->offsetGet('parameters')->willReturn(['github-pr-head' => 'sha']);
```

Comment:
```suggestion
        $activity->offsetGet('parameters')->willReturn(['github-pr-head' => 'sha', 'new_commit' => 'wrong sha']);
```

To make sure that it actually grabs the right one.